### PR TITLE
refactor: Condense `password` related files in one

### PR DESCRIPTION
**Pull request by CaedenPH**
### Describe your change:

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: other/password.py [hunk 1]
```diff
@@ -62,6 +63,45 @@ def random_characters(chars_incl, i):
...
+    'Password should contain UPPERCASE, lowercase, numbers, special characters'
+
+    >>> strong_password_detector('Hello1238udfhiaf038fajdvjjf!jaiuFhkqi1')
+    'This is a strong Password'
+
+    >>> strong_password_detector(0)
+    'Your Password must be at least 8 characters long'
+    """
+
+    if len(str(password)) < 8:
```
Comment (cclauss) [inline other/password.py:87]:
```suggestion
    if len(password) < min_length:
```
 Also, short passwords should raise ValueError.  Same with weak passwords.

**Comment by CaedenPH** [comment]
@cclauss Instead of returning a string it should raise an exception, I agree. 
However, I will go through every function after this gets merged and raise errors instead of returning strings. I'll make sure that I also update the doctests accordingly

**Comment by CaedenPH** [comment]
PS
I'll fix the failing test for other/password but the others are broken as I'm sure you're aware of

**Comment by cclauss** [comment]
Please rebase this on the current `master`

**Pull request merged**